### PR TITLE
Updated COPPA guidelines to reference the most relevant information p…

### PR DESCRIPTION
**Pull request by ragerdude**
…ertaining to the StorySquad use case with documentation from COPPA

# Description

Replace with description of PR

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update

## Change Status

- [x] Complete, tested, ready to review and merge
- [ ] Complete, but not tested (may need new tests)
- [ ] Incomplete/work-in-progress, PR is for discussion/feedback

## Has This Been Tested

- [ ] Yes
- [ ] No
- [x] Not necessary

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [x] There are no merge conflicts

### CODE REVIEW

File: data/README.md [hunk 1]
```diff
@@ -1,6 +1,13 @@
 # Story Squad Data
 
-- Children's story submissions are not able to be made public due to [COPPA](https://www.ecfr.gov/cgi-bin/text-idx?SID=4939e77c77a1a1a08c1cbf905fc4b409&node=16%3A1.0.1.3.36&rgn=div5) guidelines, and our team decided to extend that to transcriptions as well out of precaution. 
+- According to https://www.ftc.gov/tips-advice/business-center/guidance/complying-coppa-frequently-asked-questions-0#F.%20Photos, FAQ F.2 states:
```
Comment (bofori20) [inline data/README.md:3]: This helps resolve all lingering issues regarding COPPA and children's right to privacy. It helps us in gathering data from this sample group. Great job.

**Review by bofori20 — APPROVED** [review]
Like the effort put into this to resolve issues with COPPA. Now we can get the data we need to improve our model. Great job

File: data/README.md [hunk 2]
```diff
@@ -1,6 +1,13 @@
...
+
+   COPPA applies to photos, videos, and audio files that contain children’s images or voices. It also applies to geolocation data contained in these files sufficient to identify street name and name of city or town. Finally, it applies to any persistent identifiers collected via the children’s upload of their photos. Therefore, in order to offer an app without parental notice and consent, the operator must take the following steps:
+
+      1. Pre-screen the children’s photos in order to delete any that depict images of children or to delete the applicable portion of the photo, if possible. The operator must also delete any other personal information, for example, geolocation metadata, contained in the photos prior to posting them through the app. Note that if an operator does not pre-screen, then it may be subject to civil penalties under COPPA if any personal information is collected from children without the operator first notifying parents and obtaining their consent; and
```
Comment (nlittlepoole) [inline data/README.md:7]: This a great thing to flag and definitly something that should be added to the roadmap. Luckily this is actually easier to do than the OCR itself.

**Review by nlittlepoole — APPROVED** [review]
Thanks for tackling this! I think this will help clarify things for current/future cohorts

**Pull request merged**